Tests/MapboxMapsTests/Style/AttributionTests.swift

```diff
@@ -59,7 +59,9 @@ class AttributionTests: XCTestCase {
     }
 
     func testPlainTextAttributionParsing() throws {
-        let attributionString = String.randomASCII(withLength: 10)
+        var characterSet = CharacterSet(charactersIn: "©")
```

Comment:
Curious that randomASCII would emit a `©`, which is unicode `0xA9`. `randomASCII` generates characters in `0x20...0x7E`.

Reply:
Spaces are definitely possible (0x20)

Reply:
True that, removed it.

Reply:
True that, removed it.